src/styles/calendar.scss

```
@@ -0,0 +1,64 @@
+@import "./variables.scss";
+
+.calendar {
+  padding: $calendar-padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  max-width: calc(
+    7 * #{$calendar-cell-size} +
+    6 * #{$calendar-gap} +
+    2 * #{$calendar-padding}
```

```suggestion
  max-width: 7 * $calendar-cell-size + 6 * $calendar-gap + 2 * $calendar-padding
```
You don't need to use `calc`, sass are calculate everithing. 
And i think it woold be better to move this calculations to variable with name `$max-width: 7 * $calendar.........`

---

by some reason it doesn't work for me without `calc`

---

ok, my fault